VorpalEngine/sources/video/Renderer.cpp

```diff
@@ -1,40 +1,85 @@
 #include <vorpal/video/Renderer.hpp>
-
 #include <vorpal/core/Settings.hpp>
-
 #include <vorpal/utility/Logger.hpp>
+#include <vorpal/utility/asset/SimpleStorage.hpp>
 
 #include <cstring>
+#include <iostream>
+#include <set>
+#include <algorithm>
+
+//TODO temp until Dmitry will fix load code
+#include <fstream>
+static std::vector<char> readFile(const std::string& filename)
+{
+   std::ifstream file(filename, std::ios::ate | std::ios::binary);
+   if (!file.is_open()) {
+       throw std::runtime_error("Failed to open file!");
+   }
+   size_t fileSize = (size_t) file.tellg();
+   std::vector<char> buffer(fileSize);
+   file.seekg(0);
+   file.read(buffer.data(), fileSize);
+   file.close();
+   return buffer;
+}
+// TODO still temp
 
 namespace vp
 {
 namespace video
 {
 
-VkResult CreateDebugReportCallbackEXT(VkInstance instance,
-    const VkDebugReportCallbackCreateInfoEXT* pCreateInfo,
-    const VkAllocationCallbacks* pAllocator,
-    VkDebugReportCallbackEXT* pCallback)
+static VKAPI_ATTR VkBool32 VKAPI_CALL DebugCallback(VkDebugReportFlagsEXT flags,
+                                                    VkDebugReportObjectTypeEXT objType, 
```

Comment:
I don't think we should align to the right (to the opening bracket) but instead use 1 extra indent on the next lines